lambda_src/lambda_function.py

```diff
@@ -19,6 +24,27 @@
 DESTINATION_URI_HEADER = 'sf-custom-destination-uri'
 
 
+def process_row_params_format(
+    headers: Dict[Text, Text], args: Tuple[Any]
+) -> Dict[Text, Any]:
+    '''
+    Format parameters in headers with arguments from a row.
```

Comment:
a little ambivalent about this. it doesn't explain why `-` is turned to `_` and `sf-custom` is removed. maybe add that it's in preparation to be used as parameters to the call driver, or call it `process_headers_with_row`?

Reply:
Will do 👍